cdap-docs/_common/common-build.sh

```diff
@@ -257,21 +240,31 @@ function build_web() {
   build_extras
 }
 
+function build_extras() {
+  # Over-ride this function in guides where Javadocs or licenses are being built or copied.
+  # Currently performed in reference-manual
+  echo "No extras being built or copied."
+}
+
 function set_mvn_environment() {
+  cd ${PROJECT_PATH}
   if [ "$(uname)" == "Darwin" ]; then
+    # TODo: hard-coded Java version 1.7
     export JAVA_HOME=$(/usr/libexec/java_home -v 1.7)
   fi
+  # check BUILD.rst for changes
+  BUILD_RST_PATH="${PROJECT_PATH}/${BUILD_RST}"
+  test_an_include ${BUILD_RST_HASH} ${BUILD_RST_PATH}
 }
 
 function check_includes() {
   if [ "${CHECK_INCLUDES}" == "${TRUE}" ]; then
```

Comment:
Why compare string to string here? Why even have TRUE as a constant? Why not just evaluate CHECK_INCLUDES as a boolean?